src/Features/Core/Portable/AddConstructorParametersFromMembers/AddConstructorParametersFromMembersCodeRefactoringProvider.cs

```diff
@@ -181,7 +185,7 @@ static AddConstructorParametersCodeAction GetOptionalContructorParametersCodeAct
             foreach (var action in actions)
             {
                 var changedSolution = await action.GetChangedSolutionInternalAsync(
-                    priorDocument.Project.Solution, postProcessChanges: true, cancellationToken).ConfigureAwait(false);
+                    priorDocument.Project.Solution, CodeAnalysisProgress.Null, postProcessChanges: true, cancellationToken).ConfigureAwait(false);
                 Contract.ThrowIfNull(changedSolution);
```

Comment:
Unfortunately, 'intents' have no way to report progress.  It would be interesting to eventually make that work.